### ISSUE TITLE
panActivationKey does not change functionality

### ISSUE THREAD

**Issue by ccmoidel13** [xyflow/xyflow#3516]
### Describe the Bug

I am trying to use `panActivationKeyCode` and it doesn't seem to be effecting the behavior of my app. See code below: 

```
<ReactFlow
      fitView
      fitViewOptions={{ maxZoom: 1.5, duration: 300 }}
      ...
      zoomOnScroll
      zoomActivationKeyCode={null}
      panActivationKeyCode={'Meta'}
    >
```

If I do the following, it does behave as expected which means the `zoomActivationKeyCode` is working.

```
<ReactFlow
      fitView
      fitViewOptions={{ maxZoom: 1.5, duration: 300 }}
      ...
      panOnScroll
      zoomActivationKeyCode={'Meta'}
    >
```

### Your Example Website or App

https://codesandbox.io/s/gracious-easley-2x2v4j?file=/src/Flow.js

### Steps to Reproduce the Bug or Issue

1. Open app
2. scroll, zooming happens
3. press Meta and scroll, zooming happens

### Expected behavior

As a user, I would expect that when I press the Meta key the app would pan instead of zoom

### Platform

- OS: macOS
- Browser: Chrome
- Version: 117.0.

**Comment by mirisony** [comment]
I'm having the exact same issue 🙏

**Comment by moklick** [comment]
`panActivationKey` is for activating panning when `panOnDrag={false}`. If you set `panOnScroll={true}` and `panOnDrag={false}`, you can see that it works. You can only pan by dragging if "Meta" is pressed `panOnDrag={false}` https://codesandbox.io/s/vigorous-breeze-tdsds9?file=/src/Flow.js:864-940

**Comment by mirisony** [comment]
Thank you for clarifying @moklick!
In that case, is it possible to add / set a behavior which sets zoom on scroll as default, and pan on scroll scroll on key press? (the exact opposite of the panOnScroll & zoomActivationKeyCode behavior).

**Comment by bcakmakoglu** [comment]
@moklick Couldn't panning by either method be allowed when the key is activated or is that a deliberate decision?

**Comment by moklick** [comment]
This should be fixed in v11.10.1
